src/middlewared/middlewared/alert/source/volume_version.py

```diff
@@ -10,45 +9,22 @@ class VolumeVersionAlertClass(AlertClass):
     level = AlertLevel.WARNING
     title = "New Feature Flags Are Available for Pool"
     text = (
-        "New feature flags are available for volume %s. Refer "
-        "to the \"Upgrading a ZFS Pool\" subsection in the "
-        "User Guide \"Installing and Upgrading\" chapter "
-        "and \"Upgrading\" section for more instructions."
-    )
-
-
-class ZfsVersionOutOfDateAlertClass(AlertClass):
-    category = AlertCategory.STORAGE
-    level = AlertLevel.WARNING
-    title = "ZFS Filesystem Version Is Out of Date"
-    text = (
-        "ZFS filesystem version is out of date. Please consider upgrading it. See <a href=\""
-        "https://www.ixsystems.com/documentation/freenas/11.2/install.html#upgrading-a-zfs-pool\">"
+        "New ZFS version or feature flags are available for pool %s. Please see <a href=\""
+        "https://www.truenas.com/docs/hub/tasks/advanced/upgrading-pool/\">"
```

Comment:
Do ask review frm Docs team when changing text please.

Also this link is using latest docs link, perhaps it needs to be tied to 12.0 version?

Reply:
I have not changed the text, had I?

12.0 is the latest version, docs website does not give us a chance to have a 12.0-tied link.

Reply:
Actually you did =P

Either way, we should ask for their review.

@Mrt134 thoughts? My main concern is having links that will always work for 12.0.
When SCALE/12.1 are released this will stop working and/or be inaccurate?

Reply:
@Mrt134 ping

Reply:
@freenas/docs-team ping

Reply:
@Mrt134 @freenas/docs-team ping :)